### DISCUSSION TITLE
is M1 macbook supported?

### DISCUSSION THREAD

**Discussion by CKT90** [mapeditor/tiled#3461] [Q&A]
planning to buy a M1 macbook air to replace current window laptop.

checking if the M1 macbook able to run this software?

**Comment by bjorn** [comment]
[A famous guy once asked that same question](https://twitter.com/MikaelForslind/status/1346150462752485382), and it was confirmed to run. These days the releases are even signed, notarized and stapled since I decided to pay the Apple developer tax. They still rely on Rosetta though since they're not compiled for the M1 architecture.

Just yesterday though, [somebody said it wasn't working well on their M1](https://itch.io/post/6524104). I hope they will be able to provide further details.

**Reply by CKT90** [reply]
looking forward for a solid confirmation.
really hope this can work well with M1 before deciding to purchase one...
